recipes/crms2map/meta.yaml

```diff
@@ -0,0 +1,108 @@
+{% set name = "crms2map" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name |lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jinikeda/{{ name |lower }}/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 7c718dbe50dd3fe7cb2cf294418e846028126ed6a4f9c3960db92b7cfce3ae53
+
+build:
+  # Uncomment the following line if the package is pure Python and the recipe is exactly the same for all platforms.
+  # It is okay if the dependencies are not built for all platforms/versions, although selectors are still not allowed.
+  # See https://conda-forge.org/docs/maintainer/knowledge_base.html#noarch-python for more details.
+  # noarch: python
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  # More info about selectors can be found in the conda-build docs: 
+  # https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#preprocessing-selectors
+  noarch: generic
```

Comment:
This should probably be `noarch: python`

Reply:
Modified. Thank you.